### PR TITLE
aurutils: release 3.1

### LINKED ISSUE #782: aur-chroot.1: document devtools limitations
To address some common questions/pain points.
- `PKGDEST` not checked for existing packages (`makechrootpkg`)
- Signing with `makepkg --sign` not possible (https://lists.archlinux.org/pipermail/arch-projects/2016-May/004341.html)
- `Include` not respected in `pacman.conf`
- No way to disable `CacheDir`/mirror parsing of `pacman.conf`
- Hardcoded `makepkg` arguments (i.e. `--noconfirm`)

### LINKED ISSUE #784: aur-repo does not accept no arguments, contrary to man page
The man page of `aur-repo` states 
> When run without arguments, `aur-repo` prints the path of a local (file://) repository to standard output. 

This is incorrect, because running `aur-repo` without arguments gives:
> usage: repo [-d repo] [-r path] [-alpqtuS]

If I'm not mistaken, adding `-p` gives the old behavior.

For me, it is not clear whether `aur-repo`'s behavior or the man page is erroneous.

### LINKED ISSUE #785: Wrong `aur-remove` example in aur.1
The following snippet belongs to the `aur-remove` example in aur.1:
```bash
aur repo --repo-list | while read -r repo_path; do
      if repo_path=$(readlink -e "$repo_path"); then
           repo-remove "$repo_path" "$@"
           paccache -c "${repo_path%/*}" -rvk0 "$@"
       fi
done
```

This does not work as intended, because `aur repo --repo-list` only lists the _names_ of the repos, not their paths. It should probably read:
```bash
aur repo --repo-list | while read -r repo_name; do
      repo_path=$(aur repo -d "$repo_name" -p)
      repo-remove "$repo_path" "$@"
      paccache -c "${repo_path%/*}" -rvk0 "$@"
done
```
(I omitted the `readlink` step, because the manpage of `aur repo -p` states "the resolved path".)

### PR DESCRIPTION
**Pull request by AladW**
Fixes #782 
Fixes #784 
Fixes #785

### CODE REVIEW

File: man1/aur-chroot.1 [hunk 1]
```diff
@@ -260,6 +263,53 @@ QEMU user mode. See
...
+.RB ( "makepkg \-\-noconfirm" )
+.PP
+.B arch\-nspawn
+will unconditionally modify the supplied
+.BR pacman.conf (5)
+to copy mirrors (and other settings) from the host (see
+.B FS#27544
+and
+.B FS#38641
+for details.) Furthermore, repositories must be configured explicitely
```
Comment (maximbaz) [inline man1/aur-chroot.1:307]: explicitly

File: man1/aur-build.1 [hunk 2]
```diff
@@ -67,6 +69,19 @@ where <database> is specified with
...
+.B /etc/pacman.conf
+for this configuration file, as certain directives (e.g.
+.BR IgnorePkg )
+may cause issues in the container. It is recommended to instead copy
+the template in
+.B /usr/share/devtools/pacman-extra.conf
+or
+.B /usr/share/devtools/pacman-multilib.conf
+and make the necessary adjustments. Due to devtools limitations, any
+desired repositories should be configured explicitely in this copy.
```
Comment (maximbaz) [inline man1/aur-build.1:83]: explicitly (I also can never spell this word correctly...)

File: makepkg/aurutils.changelog [hunk 3]
```diff
@@ -1,3 +1,36 @@
+## 3.1.0 - 2020-11-09
+
+* `aur`
+  + fix example `aur-remove` script (#)
```
Comment (maximbaz) [inline makepkg/aurutils.changelog:4]: Don't forget to fill the issue numbers 😉

**Review by maximbaz — APPROVED** [review]
👍

**Comment by AladW** [comment]
https://thumbs.gfycat.com/LastPleasantBluefish-mobile.mp4

**Pull request merged**